### DISCUSSION TITLE
COB Led Strips

### DISCUSSION THREAD

**Discussion by dbell669** [awawa-dev/HyperHDR#658] [Q&A]
Has Anyone seen or tried to use any of those COB lights that are out? Just mainly curious about if they are worth looking into. Something like 

https://www.btf-lighting.com/collections/fcob-led-strip/products/fcob-spi-rgbw-ic-led-strip-light-ws2814-addressable-784-led-dream-color-10mm-dc24v-ip30-sk6812-high-flexible-fob-cob-lights-ra90.

I'm just wondering if they are worth looking into.  They mention a single chip being for like an 1.5 inch section but I'm not really sure if that means that segment can  only be a solid color. I'm just wondering if it would be a viable solution.

**Comment by Gittegatt** [comment]
+1 for the experience request

**Reply by dbell669** [reply]
I did some testing on this.  The lights work, but I'd wait a bit before getting them. I've had some issue finding any individually addressable ones, and most I've found seem to only take the Ws2811 Firmware, not the sk6812 one.

**Comment by evelant** [comment]
@dbell669 did you find a configuration that works well? I got some of these strips, they're really nice and bright, work well with wled. I'm just getting started trying to set up hyperhdr. https://www.aliexpress.us/item/3256805811194861.html?gatewayAdapt=glo2usa

**Reply by dbell669** [reply]
The bog standard configuration as written in the guide will work well. I think most of my issues were from poor wiring if I'm going to be honest. Just don't forget to use power injections

**Reply by evelant** [reply]
Which image did you use for the esp firmware? It's not clear to me which one will work with these ws2814 strips

**Reply by dbell669** [reply]
Apologies for not seeing this. I used Pi Picos.  Check https://github.com/awawa-dev/HyperSerialPico for the firmware.

**Comment by Skaronator** [comment]
Does anyone have these and can share whether neutral white or cold white is preferred?

**Reply by Skaronator** [reply]
I was considering buying it now to give it a try, but while the BTF Lightning FCOB has 896 LEDs per meter, it only offers 16 zones per meter. This means there would only be 12 zones on each side of my 65" TV, which doesn't seem like much.

Edit: Looks like Play TV Lightstrip from Philips only has 2,5 zones on each side:

![image](https://github.com/user-attachments/assets/7ebcd515-1373-4d35-99a1-111615f2d18a)

**Reply by dbell669** [reply]
Personally. I would go neither. I went the Philips tv route first. but like you said. its only 7 zones. I would start here:

https://www.amazon.com/stores/page/5C77B767-AD14-4012-8007-E31D2608E529?ingress=2&visitId=75a616dc-588f-4fe2-b740-79b1529318a7&store_ref=bl_ast_dp_brandLogo_sto&ref_=ast_bln

Each pixel is addressible.  This, a Pi Pico ( Firmware Here:  https://github.com/awawa-dev/HyperSerialPico) , one of the reccommended grabbers, a splitter, and a power supply and you will be good to go. Be mindful of how many LEDs you actually need. the further you are from a wall. less is more. A user from the OpenLg discord said it pretty well:
 " If the device hangs close to the wall, at a distance of approx. 10 to 20 cm, 60 LEDs per meter are sufficient. From a greater distance of between 20 and 40 cm, even 30 LEDs per meter are sufficient, as the radius of action increases."  
 
 I did 144 LED/m and to be honest it can get _really_  bright, almost illuminating the whole wall. as for your original question, Cold White is the way to go.

**Comment by dbell669** [comment]
I would try this,

https://a.aliexpress.com/_mNV7TMR

This is all individually addressable.btyere is no white channel though.
